src/InteractiveWindow/Editor/InteractiveWindow_UIThread.cs

```diff
@@ -348,7 +348,7 @@ public void AddInput(string command)
                 }
 
                 FinishCurrentSubmissionInput();
-                _window._history.Add(_window._currentLanguageBuffer.CurrentSnapshot.GetExtent());
+                _window._history.Add(_window._currentLanguageBuffer.CurrentSnapshot.GetExtent().TrimEnd());
```

Comment:
Actually, this looks pretty reasonable since the same call is made before the other ```_history.Add``` call.  If we're going to trim the end in some cases, we should trim the end in all cases.